presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java

```diff
@@ -441,6 +436,15 @@ public void destroy()
             sessionContext.getIdentity().destroy();
         }
 
+        /**
+         * Instantiates an object that contains all information about query metadata.
+         *
+         * @param token a generated token that identifies the client's request used to obtains the response from server
+         * @param uriInfo an object used to obtains the URI where query can be retrieved in server or construct the query representation inside
+         * the web interface
+         * @param dispatchInfo an object responsible for taking the coordinator's location, response time and waiting time
+         * @return an object with all query's metadata
```

Comment:
could reference a {@link QueryResults} in the return statement

Reply:
https://softwareengineering.stackexchange.com/questions/199293/is-repeating-links-to-the-same-class-in-a-single-javadoc-comment-a-bad-practice